src/features/create-dao/CreateDAOHeader.tsx

```diff
@@ -0,0 +1,48 @@
+//- React Imports
+import { FC, useContext } from 'react';
+
+//- Style Imports
+import styles from './CreateDAOHeader.module.scss';
+
+//- Component Imports
+import { IconCross } from '@zero-tech/zui/components/Icons';
+import { StepBar } from '@zero-tech/zui/components/StepBar/StepBar';
+import { Wizard } from '@zero-tech/zui/components/Wizard/Wizard';
```

Comment:
nit: should be able to reduce these imports down to one line, similar to comments around imports above